tests/__init__.py

```diff
@@ -24,7 +24,7 @@
 # yield identical MusicXML
 MUSICXML_IMPORT_EXPORT_TESTFILES = [
     os.path.join(MUSICXML_PATH, fn)
-    for fn in ["test_note_ties.xml", "test_note_ties_divs.xml"]
+    for fn in ["test_note_ties.xml", "test_note_ties_divs.xml", "test_score_object.musicxml"]
```

Comment:
This is a minor comment, but from the tests I see that you added a piece to that list, but then you index the pieces in the list to not include it. My question is do the test fail when you include it? But if is only useful for a single test then is should not be grouped with the others.

Reply:
Both tests (`test_reconstruction_score()` in `test_pianoroll.py` and `test_import_export()` in `test_xml.py` fail if the additional test file is included. The third test file in the above list (`test_score_object.musicxml`) is now accessed separately.

Reply:
Why are the tests failing? Shouldn't they go through if the file is valid.

Reply:
I didn't look further into the functions, and the file itself was already a test file, I only added meta information.